lib/pbench/server/api/resources/users_api.py

```diff
@@ -321,15 +307,17 @@ def get(self, username):
             Authorization:  Bearer Pbench_auth_token (user received upon login)
 
         :return: JSON Payload
-            response_object = {
-                    "message": "Success"/"failure message",
-                    "data": {
+            Success: 200,
+                    response_object = {
```

Comment:
I'm confused. I still don't have "hide whitespace changes" enabled in GitHub, but it looks like you _increased_ the indentation on line 311 to match the now-overly-indented lines 312 to 314, whereas I'd have expected the opposite.

It's a docstring comment; I'm not overly concerned unless GitHub is lying to me...